e2e-tests/charge-cycles-on-reject.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+set -Eexuo pipefail
+
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+source "${SCRIPT_DIR}/utils.sh"
+
+# Run dfx stop if we run into errors.
+trap "dfx stop" EXIT SIGINT
+
+dfx start --background --clean
+
+# Deploy the canister that returns the blocks for charge-cycles-on-reject.
```

Comment:
I'm not sure what this means. We are not fetching blocks from this canister at all, right? Isn't the reason here that we want to send the `send_transaction` endpoint? If so, can you clarify that in the comment?

Reply:
You are right, I have changed that.